tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -201,6 +166,21 @@
     import {setRouter, createRouter} from "../tf-backend/router.js";
     import {fetchRuns} from "../tf-backend/runsStore.js";
 
+    const COMPONENTS = {
+      'scalars': 'tf-scalar-dashboard',
+      'images': 'tf-image-dashboard',
+      'audio': 'tf-audio-dashboard',
+      'graphs': 'tf-graph-dashboard',
+      'distributions': 'tf-distribution-dashboard',
+      'histograms': 'tf-histogram-dashboard',
+      'embeddings': 'vz-projector-dashboard',
+      'text': 'tf-text-dashboard',
+    };
+    if (!_.isEqual(Object.keys(COMPONENTS), TABS)) {
+      throw new Error(
+        `Bad set of components: ${Object.keys(COMPONENTS)} vs. ${TABS}`);
```

Comment:
We could move `COMPONENTS` to globals. I opted not to do that with the intention of making the globals file Polymer-blind.

I think what we'll have Soon™ is a file `tf-components-list.html` that (only) provides this `COMPONENTS`, where this file is generated at build time by a skylark rule. That file could also provide `TABS` if we want.